src/app/globals.css

```
@@ -163,6 +163,33 @@ body {
     color: var(--text-secondary);
   }
 
+  /* Tabs (agent page) */
+  .tab {
+    @apply -mb-px inline-flex items-center gap-2 border-b-2 border-transparent px-3 py-2.5 text-sm font-medium transition-colors;
+    color: var(--text-secondary);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)